### ISSUE TITLE
Move away from Keynote

### ISSUE THREAD

**Issue by nikomatsakis** [nikomatsakis/intorust#1]
I'd love to move away from Keynote slides, I'm just not sure what to move _to_.

Some requirements:
- Able to easily make graphics and pictures
- Able to easily get onto the web
  - Along with screencasts, I'd like to be able to step through the slides and see notes and so forth
- Able to have people edit and submit PRs
- Easy to edit

Some nice-to-haves:
- Ability to do animations
  - it might be can workaround this by just having things appear in different places

Some candidates:
- reveal.js by itself isn't that great, but the [mon-artise](https://pnkfelix.github.io/mon-artiste/demo.html) project has potential

**Comment by ests** [comment]
Well, my first though was reveal.js actually.
Besides that, is the "fire" animation obligatory? ;)

**Comment by nikomatsakis** [comment]
@ests 

>  Besides that, is the "fire" animation obligatory? ;)

Yes. It's my trademark! =) More seriously, though, I do think that animations are helpful. It's quite possible though that a careful sequence of diagrams (maybe supplemented with CSS transitions) can more than fill the gap.

To be honest I am not inclined to revisit the *existing* slides so much, at least not *now*, just because the current screencasts are good and I would rather work on new materials. I was thinking of doing some work on a tutorial covering structs-and-enums (since it seems like such a core foundation), and I had the thought of trying to reproduce said slides in mon-artiste or something similar.

cc @pnkfelix

**Comment by ctjhoa** [comment]
For what it's worth, I used https://slides.com/ in the past and it's a good tool too (powered by reveal.js).

**Comment by nixpulvis** [comment]
Maybe LaTeX? Or too much syntactic overhead?
